lib/block-patterns.php

```diff
@@ -70,3 +70,86 @@
 						<!-- /wp:query -->',
 	)
 );
+
+register_block_pattern(
+	'query/grid-posts',
+	array(
+		'title'      => __( 'Grid', 'gutenberg' ),
+		'blockTypes' => array( 'core/query' ),
+		'categories' => array( 'Query' ),
+		'content'    => '<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"layout":{"type":"flex","columns":3}} -->
+						<!-- wp:query-loop -->
+						<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"layout":{"inherit":false}} -->
+						<main class="wp-block-group" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px"><!-- wp:post-title {"isLink":true} /-->
+						<!-- wp:post-excerpt /-->
+						<!-- wp:post-date /--></div>
+						<!-- /wp:group -->
+						<!-- /wp:query-loop -->
+						<!-- /wp:query -->',
+	)
+);
+
+register_block_pattern(
+	'query/large-title-posts',
+	array(
+		'title'      => __( 'Large Title', 'gutenberg' ),
+		'blockTypes' => array( 'core/query' ),
+		'categories' => array( 'Query' ),
+		'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"100px","right":"100px","bottom":"100px","left":"100px"}},"color":{"text":"#ffffff","background":"#000000"}}} -->
+						<div class="wp-block-group alignfull has-text-color has-background" style="background-color:#000000;color:#ffffff;padding-top:100px;padding-right:100px;padding-bottom:100px;padding-left:100px"><!-- wp:query {"queryId":3,"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
```

Comment:
Remove `queryId` from here as well.

Reply:
Thanks for catching that. Fixed in https://github.com/WordPress/gutenberg/pull/30763/commits/97ff5e9d716a60d25b8a71de8e6dec4100d904cd.